css/style.css

```diff
@@ -3,3 +3,74 @@
 *::after {
   box-sizing: border-box;
 }
+
```

Comment:
Your project needs some bit of adjustments to match the provide [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). Kindly bear in mind that we actually want our design to as much as possible to follow the provided [template design](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). Kindly ensure to improve upon the following:
- [ ] The _Remove button_ is **aligned to the right** of each row (even for desktop version).
- [ ] There is **no vertical border** between each _row contents_ and the _Remove button_.
- [ ] The _Add button_ is **aligned to the right** with the form inputs.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/134490353-6132b038-d121-4cf6-b857-a9315b3978c3.png)


**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/134490421-44d9aed2-24ad-4801-9fd9-10c1f43ebb5e.png)

Reply:
Hello Team, kindly note that the above is a requirement. I mistakingly approved your project. Please attempt to implement the said changes. I'm therefore changing it to an optional requirement because it is approved. However, please attempt to implement the said changes